.travis.yml

```diff
@@ -91,7 +91,7 @@ install:
 
 script:
   - | # Deploy PR to staging environment
```

Comment:
Nit: a comment would be great. (e.g. "... only when Travis secrets are available")